### PR TITLE
Improve issue templates

### PR DESCRIPTION
**Pull request by Zero3141**
## Description
Fixes [#1352](https://github.com/secureCodeBox/secureCodeBox/issues/1352) by adjusting the "⚓️ New Hook request" and "🚓 New Security Scanner request" issue templates

### Checklist

* [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] Make sure that all your commits are signed-off and that you are added to the [Contributors](https://github.com/secureCodeBox/secureCodeBox/blob/main/CONTRIBUTORS.md) file.
* [ ] Make sure that all CI finish successfully.
* [ ] Optional (but appreciated): Make sure that all commits are [Verified](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_hook.md [hunk 1]
```diff
@@ -31,11 +31,10 @@ Before opening a new issue, please make sure that we do not have any duplicates
...
 Hint: A general guide how to implement a new SCB Hook is documented [here](https://www.securecodebox.io/docs/contributing/integrating-a-hook)
 
-- [ ] Create a new folder with the name of the [hook here](https://github.com/secureCodeBox/secureCodeBox/tree/master/hooks)
-- [ ] Add a `README.md` and give a brief overview of the scanner and its configuration options.
-- [ ] Add a HelmChart and document all configuration options.
-- [ ] Add (optional) a `Dockerfile` for the scanner if there is no existing one publicly available on dockerHub.io
-- [ ] Use the [Hook-SDK](https://github.com/secureCodeBox/secureCodeBox/tree/master/hook-sdk) to implement a new hook (currently based on NodeJS)
-- [ ] Add unit tests with at minimum 80% test coverage
-- [ ] Add some example `scan.yaml` and `finding.yaml` files in the example folder
+- [ ] Create a new Helm Chart with the `helm create new-hook` command (replace `new-hook` with the name of the hook)
```
Comment (Ilyesbdlala) [inline .github/ISSUE_TEMPLATE/new_hook.md:34]: Maybe add that the command have to be run in hooks folder ?

File: .github/ISSUE_TEMPLATE/new_security_scanner.md [hunk 2]
```diff
@@ -31,14 +31,13 @@ Before opening a new issue, please make sure that we do not have any duplicates
...
 
-- [ ] Create a new folder with the name of the [scanner here](https://github.com/secureCodeBox/secureCodeBox/tree/master/scanners)
-- [ ] Add a `README.gotmpl` and give a [brief overview](https://www.securecodebox.io/docs/contributing/integrating-a-scanner/readme) of the scanner and its configuration options.
-- [ ] Add a HelmChart and document all configuration options.
-- [ ] Implement a new scanner specific `scan-type.yaml`
-- [ ] Implement a new scanner specific `parse-definition.yaml`
-- [ ] Add (optional) some `cascading-rules.yaml` like documented [here](https://www.securecodebox.io/docs/api/crds/cascading-rule)
-- [ ] Add (optional) a `Dockerfile` for the scanner if there is no existing one publicly available on dockerHub
-- [ ] Use the [parser-SDK](https://github.com/secureCodeBox/secureCodeBox/tree/master/parser-sdk) to implement a new findings parser (currently based on NodeJS)
+- [ ] Create a new Helm Chart with the `make create-new-scanner NAME=new-scanner` command (replace `new-scanner` with the name of the scanner)
```
Comment (Ilyesbdlala) [inline .github/ISSUE_TEMPLATE/new_security_scanner.md:34]: Also say here from where the command should be run

**Review by Ilyesbdlala — APPROVED** [review]
LGTM

**Review by Ilyesbdlala — APPROVED** [review]
